### DISCUSSION TITLE
Editor Team Question: Controls and Shortcuts

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#4] [Q&A]
Does it feel intuitive? What could be adjusted? What did you try to do vs. how it is currently implemented?

**Comment by Hatchibombotar** [comment]
Maybe an option to use Q and E to move the character up/down, separate from normal (cross-hair mode) controls.

**Reply by minecraftchloe** [reply]
To confirm, this is Tool Mode -> right mouse button held down -> replacing shift/space and would be the mapping for non-QWERTY keyboard?

**Comment by StarkTMA** [comment]
The movement controls do not inherit from the base game controls, WASD isn't practical for non-QWERTY keyboards.

**Reply by minecraftchloe** [reply]
Agree on this - fantastic feedback for how we can make this more inclusive!

**Reply by minecraftchloe** [reply]
We made an update that defers to the base game controls. What do you think?

**Comment by ihategravel2** [comment]
CTRL + TAB to enter Crosshair Mode is a pretty awkward key combination to press. I think a  a single button like "f" to easily go back and forth between Editor and Crosshair Mode would be better.  I'm unaware of any control that also uses the "F" key.

**Reply by CaptainCooky** [reply]
I find the Ctrl+Tab to be a good combination. 
That said, we're all different, so having the ability to assign custom Keybinds would keep us both happy.

**Reply by ihategravel2** [reply]
Yes, keyboards would be nice.

**Reply by ihategravel2** [reply]
*keybinds*

**Comment by Marceloni** [comment]
I'd like being able to select text in text fields to easily edit and copy paste coordinates for example. I know that CTRL + A works to select everything but it would be nice to select a part of the text with the mouse. Also there's currently no visual indication that you selected something.

**Reply by minecraftchloe** [reply]
Definitely we have some improvements to make on the input/textfield usability! This is great feedback, thank you!
